projects/angular/src/progress/progress-bars/_progress-bars.clarity.scss

```diff
@@ -618,13 +670,51 @@
     align-items: center;
     justify-content: center;
 
+    @include utils-mixins.css-var(color, clr-progress-label-color, inherit, variables.$clr-use-custom-properties);
+    @include utils-mixins.css-var(
+      font-weight,
+      clr-progress-standart-font-weight,
+      inherit,
+      variables.$clr-use-custom-properties
+    );
+    @include utils-mixins.css-var(
+      font-size,
+      clr-progress-label-font-size,
+      inherit,
+      variables.$clr-use-custom-properties
+    );
+    @include utils-mixins.css-var(
+      line-height,
+      clr-progress-label-line-height,
+      inherit,
+      variables.$clr-use-custom-properties
+    );
+    @include utils-mixins.css-var(
+      letter-spacing,
+      clr-progress-label-letter-spacing,
+      inherit,
+      variables.$clr-use-custom-properties
+    );
+
     & > * {
       flex: 0 0 auto;
-      padding-right: variables.$clr_baselineRem_0_5;
+      @include utils-mixins.css-var(
+        padding-right,
+        clr-progress-block-items-padding-right,
+        variables.$clr_baselineRem_0_5,
+        variables.$clr-use-custom-properties
+      );
       // vertically center???
 
+      // ATM (after token migration)
+      // remove first child padding right
       &:first-child {
-        padding-right: variables.$clr_baselineRem_0_75;
+        @include utils-mixins.css-var(
+          padding-right,
+          clr-progress-block-first-item-padding-right,
```

Comment:
Seems right but I see we give / have examples and that's the label. Should we name it `clr-progress-block-lable-padding` or keep it this way?

Reply:
`clr-progress-block-first-item-padding-right` will be removed after token migration. All children will have same padding right except the last one. This property is temp solution so we don't break BG theme.